src/hooks/useAlert.ts

```diff
@@ -0,0 +1,25 @@
+import { useDispatch } from 'react-redux';
+
+import type { AppDispatch } from '../common/store';
+import { clearAlert, setAlert } from '../reducers/AlertSlice';
+import type { AlertType } from '../types/state';
+
+const useAlert = () => {
```

Comment:
optional comment: is this hook really useful? I feel like it's adding more code than it removes.

Reply:
I guess I don't have a strong argument for it now, but ideally in the future we will be able to display more than one alert at a time, and this hook would be used to handle those actions.